__tests__/secret.test.js

```diff
@@ -570,6 +570,18 @@ describe('Tozny identity client', () => {
     )
     expect(sharedList[0].meta.recordId).toBe(recordView.meta.recordId)
   })
+  it('can delete an unshared secret', async () => {
```

Comment:
but it does work for shared secrets as well?

Reply:
another test is coming for shared secrets -- but yes it does work for them!